### DISCUSSION TITLE
5B4B661F-69E7-BCED-AEEC-AB509A597B2B

### DISCUSSION THREAD

**Discussion by giscus** [celestiamobile/comments#22] [General]
# 5B4B661F-69E7-BCED-AEEC-AB509A597B2B

Kepler-20 is a binary star system (comprising of a Sun-like star and a red dwarf) about 929.14 light-years away in the constellation Lyra. The primary star has a total of six known planets (1 superterran, 2 terrans, 3 Neptunians)

https://celestia.mobi/addon?item=5B4B661F-69E7-BCED-AEEC-AB509A597B2B

**Comment by exocritic5457** [comment]
simply wonderful.

**Reply by exocritic5457** [reply]
....besides Ag being unsure about if it wants to actually exist or not, but it's fine
